exercises/practice/connect/connect_test.rb

```diff
@@ -11,7 +11,7 @@ def test_an_empty_board_has_no_winner
       '  . . . . .',
       '   . . . . .',
       '    . . . . .'
-    ].map {|row| row.gsub(/^ */, '')}
+    ].map { |row| row.gsub(/^ */, '') }
```

Comment:
An off-topic remark: I found the way this is written using the map/gsub combination really hard to parse as someone newer to Ruby

Reply:
@ErikSchierboom do you have a way that you would write this instead?  (Getting ready to merge this, pending the failed CI corrections).